utils/products.js

```diff
@@ -1,152 +1,664 @@
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Semantic Layer & Digital Twin Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Item Relationship Service Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
   },
-
   {
-    productName: "Item Relationship Service",
-    productDescription: "The project provides a service for ad-hoc data chains across n-tier values chains for different use cases based on the EDC.",
+    productName: "knowledge-agents",
+    productDescription:
+      "Reference Implementations of CX-0084 (Federated Queries In Data Spaces)",
+    githubRepo: ["https://github.com/eclipse-tractusx/knowledge-agents"],
+    committers: ["https://github.com/almadigabor"],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request knowledge-agents Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "knowledge-agents-aas-bridge",
+    productDescription: "TBD",
     githubRepo: [
-      "https://github.com/eclipse-tractusx/item-relationship-service",
+      "https://github.com/eclipse-tractusx/knowledge-agents-aas-bridge",
+    ],
+    committers: ["https://github.com/almadigabor"],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request knowledge-agents-aas-bridge Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "knowledge-agents-edc",
+    productDescription:
+      "EDC Extensions for CX-0084 (Federated Queries In Data Spaces)",
+    githubRepo: ["https://github.com/eclipse-tractusx/knowledge-agents-edc"],
+    committers: ["https://github.com/almadigabor"],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request knowledge-agents-edc Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "managed-identity-wallet",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/managed-identity-wallet"],
+    committers: [
+      "https://github.com/borisrizov-zf",
+      "https://github.com/nitin-vavdiya",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-identity-wallet Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "managed-service-orchestrator",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/managed-service-orchestrator",
     ],
     committers: [
-      "https://github.com/ds-jkreutzfeld",
-      "https://github.com/ds-jhartmann",
-      "https://github.com/ds-mkanal",
-      "https://github.com/jzbmw"
+      "https://github.com/almadigabor",
+      "https://github.com/tomaszbarwicki",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Item Relationship Service Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-service-orchestrator Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
+  },
+  {
+    productName: "managed-simple-data-exchanger",
```

Comment:
`managed-simple-data-exchanger`, `managed-simple-data-exchanger-backend`, `managed-simple-data-exchanger-frontend` are the same product, just different components.